data/ticket/src/main/java/jp/kztproject/rewardedtodo/data/ticket/TicketRepository.kt

```diff
@@ -1,38 +1,49 @@
 package jp.kztproject.rewardedtodo.data.ticket
 
-import android.content.SharedPreferences
+import androidx.datastore.core.DataStore
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/5a4d185363437052ce1ba6d68907f9cfb835bb6e/data/ticket/src/main/java/jp/kztproject/rewardedtodo/data/ticket/TicketRepository.kt#L3"></span>Imports must be ordered in lexicographic order without any empty lines in-between</td>
    </tr>
  </tbody>
</table>